test/fixtures/schemas.js

```diff
@@ -7,7 +7,7 @@ export const address = schema('address', {
   relationships: {
     company: belongsTo('company', 'address'),
   },
-  meta: {
+  _meta: {
```

Comment:
In schema definitions, meta information should still reside under the `meta` key, not `_meta` key. The underscore is for internal use, and the end user shouldn't be forced to use underscores.